Ref/Top/topology.fpp

```diff
@@ -92,28 +92,29 @@ module Ref {
     # ----------------------------------------------------------------------
 
     connections Downlink {
-      dpCat.fileOut -> fileDownlink.SendFile
+      # Data Products
+      dpCat.fileOut             -> fileDownlink.SendFile
       fileDownlink.FileComplete -> dpCat.fileDone
-
-      eventLogger.PktSend -> comQueue.comPacketQueueIn[Ports_ComPacketQueue.EVENTS]
-      tlmSend.PktSend -> comQueue.comPacketQueueIn[Ports_ComPacketQueue.TELEMETRY]
-      fileDownlink.bufferSendOut -> comQueue.bufferQueueIn[0]
+      # Inputs to ComQueue (events, telemetry, file)
+      eventLogger.PktSend         -> comQueue.comPacketQueueIn[0]
+      tlmSend.PktSend             -> comQueue.comPacketQueueIn[1]
+      fileDownlink.bufferSendOut  -> comQueue.bufferQueueIn[0]
       comQueue.bufferReturnOut[0] -> fileDownlink.bufferReturn
-
-      comQueue.queueSend -> fprimeFramer.dataIn
+      # ComQueue <-> Framer
+      comQueue.queueSend   -> fprimeFramer.dataIn
       fprimeFramer.dataReturnOut -> comQueue.bufferReturnIn
-      fprimeFramer.comStatusOut -> comQueue.comStatusIn
-
-      fprimeFramer.bufferAllocate -> commsBufferManager.bufferGetCallee
+      fprimeFramer.comStatusOut  -> comQueue.comStatusIn
+      # Buffer Management for Framer
+      fprimeFramer.bufferAllocate   -> commsBufferManager.bufferGetCallee
       fprimeFramer.bufferDeallocate -> commsBufferManager.bufferSendIn
-
-      fprimeFramer.dataOut -> comStub.comDataIn
+      # Framer <-> ComStub
+      fprimeFramer.dataOut        -> comStub.comDataIn
       comStub.dataReturnOut -> fprimeFramer.dataReturnIn
```

Comment:
Thoughts on naming these ports `dataReturn` vs `bufferReturn` ?
In the first PR I was using `dataReturn` when dealing with `Svc.ComDataWithContext` and `bufferReturn` when dealing with `Fw.Buffer`, and trying to match the name of the associated input port (e.g. `dataIn` input would lead to a `dataReturn`) 
But then there's also `ByteStreamData` which also uses the name `dataReturn` and it gets a little more messy than I'd like.
Maybe consolidate **everything** on `bufferReturn` regardless of input type / input port name, and definitively call this the "buffer return" pattern?

Reply:
I also like to be explicit, maybe `bufferOwnershipReturn`.. but that's quite a mouthful...

Reply:
Agreed to use the same name as the port that came in, using `ReturnIn`/`ReturnOut` suffix